WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -214,7 +214,7 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 
 			var registeredAliceClients = registeredAliceClientAndCircuits.Select(x => x.AliceClient).ToImmutableArray();
 
-			var outputTxOuts = await ProceedWithOutputRegistrationPhaseAsync(roundId, registeredAliceClients, cancellationToken).ConfigureAwait(false);
+			var (goForBlameRound, outputTxOuts) = await ProceedWithOutputRegistrationPhaseAsync(roundId, registeredAliceClients, cancellationToken).ConfigureAwait(false);
```

Comment:
The naming is confusing. The information is `maliciousInputDetected` then `shouldRegisterOutput` and now it is `goForBlameRound`. I would use `maliciousInputDetected` everywhere.